team-telnyx/webrtc
# Update react-client interface imports

Updates react-client to use interfaces exported from `@telnyx/webrtc`.

Important: Manual testing is based off the currently unreleased version on main. This PR should be merged in after publishing the latest from `@telnyx/webrtc`.

## 📝 To Do


## ✋ Manual testing

1. From `packages/js` run `npm link`
2. From `packages/react-client` run `npm link @telnyx/webrtc`
3. From `packages/react-client` run `npm build`
4. From `packages/react-client/example` run `npm link @telnyx/webrtc`
5. Set up `packages/react-client/example` and run `npm start`
6. Make a call. Verify app works as expected

## 🦊 Browser testing

### Desktop
